app/controllers/api/v1/locations_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Api
+  module V1
+    class LocationsController < ApplicationController
+      before_action :set_location, only: [:update, :destroy]
+
+      def create
+        @school = CodeSchool.find(params[:code_school_id])
```

Comment:
Perhaps use `.find_by` instead of `find` here?  That way, if a bad `code_school_id` were passed, `@school` would equal `nil`, instead of the method blowing up.

Reply:
good idea